src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -57,11 +59,12 @@ def create_job_from_job_bundle(
     max_retries_per_task: Optional[int] = None,
     print_function_callback: Callable[[str], None] = lambda msg: None,
     decide_cancel_submission_callback: Callable[
-        [dict[str, int], int, int], bool
-    ] = lambda paths, files, bytes: False,
+        [AssetUploadGroup], bool
+    ] = lambda upload_group: False,
```

Comment:
This is a backwards-incompatible change, since it changes the accepted values for this parameter, but it looks low-risk to me. Can you change your commit to be `fix!` instead of `fix` to reflect that?

Reply:
Done!